data/gtk/help-overlay.ui

```diff
@@ -32,6 +32,18 @@
                 <property name="action-name">win.open-file</property>
               </object>
             </child>
+            <child>
+              <object class="GtkShortcutsShortcut">
+                <property name="title" translatable="yes" context="shortcut window">Pause/Play</property>
+                <property name="accelerator">space</property>
+              </object>
+            </child>
+            <child>
+              <object class="GtkShortcutsShortcut">
```

Comment:
Suggestions:

- Rename the titles "Go back 10 seconds" and "Go forward 10 seconds"
- Create different shortcuts for Left and Right, instead of grouping them together